### ISSUE TITLE
Support for phrase based similarity

### ISSUE THREAD

**Issue by bhanga-manish** [amansrivastava17/siamese-text-similarity#4]
Hi,

Can you include support for phrase matching / similarity based upon character level embedding?

**Comment by amansrivastava17** [comment]
Yes we can, you can try if you want to add that feature, Will be happy to merge your pull request.
Else I will be adding it very soon.
